stages/org.osbuild.yum.config

```diff
@@ -26,13 +28,70 @@ SCHEMA = r"""
         "description": "Determines how upstream HTTP caches are instructed to handle any HTTP downloads that YUM does."
       }
     }
+  },
+  "plugins": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "YUM plugins configuration",
+    "properties": {
+      "langpacks": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "'langpacks' YUM plugin configuration",
+        "minProperties": 1,
+        "properties": {
+          "locales": {
+            "type": "array",
+            "description": "list of locales for YUM",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
   }
 }
 """
 
 
+def configure_plugins(tree, plugins_options):
+    for plugin, plugin_options in plugins_options.items():
+        plugin_conf_path = f"{tree}/etc/yum/pluginconf.d/{plugin}.conf"
+        plugin_conf = iniparse.SafeConfigParser()
+
+        try:
+            with open(plugin_conf_path, "r") as f:
+                plugin_conf.readfp(f)
+        except FileNotFoundError:
+            print(f"Warning: {plugin} configuration file '{plugin_conf_path}' does not exist, will create it.")
+
+        # ensure that the pluginconf directory exists
+        os.makedirs(f"{tree}/etc/yum/pluginconf.d/", exist_ok=True)
+
+        for option, value in plugin_options.items():
+            if option == "locales":
+                if not plugin_conf.has_section("main"):
+                    plugin_conf.add_section("main")
+
+                locales = ", ".join(value)
```

Comment:
I just realized that `locales` may end up being empty if the provided array is also empty. Would that be a problem? If yes, then the schema should state also `minItems` for `locales`. Sorry for not spotting this earlier...

Reply:
I added `minItems` for now. If we ever need to push empty locales, we can always lift the constraint.